clients/rospy/src/rospy/impl/tcpros_base.py

```diff
@@ -744,9 +744,9 @@ def receive_once(self):
         except DeserializationError as e:
             rospyerr(traceback.format_exc())            
             raise TransportException("receive_once[%s]: DeserializationError %s"%(self.name, str(e)))
-        except TransportTerminated as e:
+        except TransportTerminated:
             raise #reraise
```

Comment:
While you're here, remove the unnecessary comment, please.

Reply:
Can't, I deleted my fork. Merge this for now, I'll do it from my other PR.